### DISCUSSION TITLE
`create-vite` vs `create-vue` for scaffolding a VueJS project

### DISCUSSION THREAD

**Discussion by SayakMukhopadhyay** [vuejs/docs#2047] [Q&A] [answered]
I have noticed that the [VueJS documentation](https://vuejs.org/guide/scaling-up/tooling.html#project-scaffolding) recommends using `create-vue` to scaffold whereas the [Vite docs](https://vitejs.dev/guide/#scaffolding-your-first-vite-project) recommend using `create-vite` with the vue template.

To make matters more confusing, the [Vuetify docs](https://next.vuetifyjs.com/en/getting-started/installation/#vite) also recommend using `create-vite`.

What should someone do? Are there separate use cases for each of them?

I have opened the same [discussion](https://github.com/vitejs/vite/discussions/10483) in vite repo too.

**Comment by splincool** [comment, marked as answer]
In Vite repo discussion @patak-dev has answered that we should use `create-vue`
